email_alert_service/services.rb

```diff
@@ -0,0 +1,8 @@
+class Services
+  def self.email_api_client
+    GdsApi::EmailAlertApi.new(
```

Comment:
My knowledge of Ruby is still quite thin, but would this be better as a module, rather than a class?

Also, I realise that this has just been moved, but it might be good to cache the object here (the standard `@_email_api_client ||=)`.

Reply:
fixed it